{"name": "Вход с неверными данными → 401", "status": "passed", "steps": [{"name": "Login user", "status": "passed", "parameters": [{"name": "email", "value": "''"}, {"name": "password", "value": "''"}], "start": 1750777949232, "stop": 1750777949322}], "parameters": [{"name": "email", "value": "''"}, {"name": "password", "value": "''"}], "start": 1750777949232, "stop": 1750777949322, "uuid": "97185789-a684-4d49-9a86-74890499eca9", "historyId": "5da0defd295c5b73991b1ce5dbbe7dd0", "testCaseId": "5da0defd295c5b73991b1ce5dbbe7dd0", "fullName": "tests.test_login.TestLogin#test_login_invalid", "labels": [{"name": "story", "value": "Авторизация пользователя"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-47CL42P"}, {"name": "thread", "value": "19376-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login"}]}